docker-hub/api/latest.yaml

```diff
@@ -1135,6 +1162,60 @@ paths:
           type: string
       tags: [audit-logs]
 
+  /v2/orgs/{name}/settings:
+    parameters:
+      - in: path
+        name: name
+        description: Name of the organization.
+        required: true
+        type: string
+    get:
+      summary: Get organization settings
+      description: |
+        Returns organization settings by name.
+      tags: [org-settings]
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: "#/definitions/orgSettings"
+        401:
+          $ref: "#/responses/Unauthorized"
+        403:
+          $ref: "#/responses/Forbidden"
+        404:
+          $ref: "#/responses/NotFound"
+    put:
+      summary: Update organization settings
+      description: |
+        Updates an organization's settings. Some settings are only used when the organization is on a business plan.
```

Comment:
Updates an organization's settings.  When the organization is on a business plan, it uses specific settings.

Reply:
Per our discussion, updating to list settings that are on a business plan 😄